project/Doc.scala

```diff
@@ -139,17 +141,15 @@ object UnidocRoot extends AutoPlugin {
 object Unidoc extends AutoPlugin {
 
   override def trigger = allRequirements
-  override def requires = plugins.JvmPlugin
+  override def requires = UnidocRoot.CliOptions.genjavadocEnabled.ifTrue(GenJavadocPlugin).getOrElse(plugins.JvmPlugin)
 
   override lazy val projectSettings = UnidocRoot.CliOptions.genjavadocEnabled.ifTrue(
-    genjavadocExtraSettings ++ Seq(
+    Seq(
       javacOptions in compile += "-Xdoclint:none",
       javacOptions in test += "-Xdoclint:none",
       javacOptions in doc += "-Xdoclint:none",
       scalacOptions in Compile += "-P:genjavadoc:fabricateParams=true",
-      unidocGenjavadocVersion in Global := "0.10",
-      // FIXME: see https://github.com/akka/akka-http/issues/230
```

Comment:
This doesn't fix that issue, it merely allows to remove one of the two filters that was added as a work around/

See https://github.com/akka/akka-http/blob/b825f3e9f2495d6d1562c8b504a3a07ab375782a/project/Doc.scala#L125